Index: client/src/components/requestsList.vue
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/client/src/components/requestsList.vue b/client/src/components/requestsList.vue
--- a/client/src/components/requestsList.vue	
+++ b/client/src/components/requestsList.vue	
@@ -1,94 +1,3 @@
-<!--  <v-expansion-panels>-->
-<!--    <v-expansion-panel-->
-<!--      v-for="(request, id) in requests"-->
-<!--      :key="id"-->
-<!--    >-->
-<!--      <v-expansion-panel-header class="element-0">-->
-
-<!--        <div class="element-1"> {{ request.current_amount }} {{ request.current_currency }} {{-->
-<!--            request.current_type-->
-<!--          }}-->
-<!--        </div>-->
-<!--        <div class="element-1"> {{ request.wanted_amount }} {{ request.wanted_currency }} {{-->
-<!--            request.wanted_type-->
-<!--          }}-->
-<!--        </div>-->
-
-<!--      </v-expansion-panel-header>-->
-<!--      <v-expansion-panel-content>-->
-<div class="element-1"><strong> {{ request.id }} </strong></div>
-<div class="element-2" v-if="isCurrentUser(request.maker_username)"> Создатель заявки: Я</div>
-<div class="element-2" v-else><strong>Создатель заявки:</strong> {{ request.maker_username }} Рейтинг:
-{{ request.maker_rank }} <br>ID: {{ request.maker_id }}
-</div>
-
-<!--        <div class="element-2"><strong> Вы получите: </strong> {{ request.current_amount }} {{-->
-<!--            request.current_currency-->
-<!--          }} {{ request.current_type }} {{ request.current_country }} {{ request.current_city }} {{-->
-<!--            request.current_bank-->
-<!--          }} {{ request.current_purpose }}-->
-<!--        </div>-->
-
-<!--        &lt;!&ndash;                <div class="element-2"><strong>Вы получите: </strong></div>&ndash;&gt;-->
-<!--        &lt;!&ndash;                <div class="element-2"> {{ request.current_amount }} {{ request.current_currency }}</div>&ndash;&gt;-->
-<!--        &lt;!&ndash;                <div class="element-2"> {{ request.current_type }}</div>&ndash;&gt;-->
-<!--        &lt;!&ndash;                <div class="element-2"> {{ request.current_country }} {{ request.current_city }}</div>&ndash;&gt;-->
-<!--        &lt;!&ndash;                <div class="element-2"> {{ request.current_bank }} {{ request.current_purpose }}</div>&ndash;&gt;-->
-<!--        <div class="element-2"><strong> Вы отдаете: </strong> {{ request.wanted_amount }} {{ request.wanted_currency }}-->
-<!--          {{ request.wanted_type }} {{ request.wanted_country }} {{ request.wanted_city }} {{ request.wanted_bank }}-->
-<!--          {{ request.wanted_purpose }}-->
-<!--        </div>-->
-<!--        &lt;!&ndash;                <div class="element-2"> {{ request.wanted_amount }} {{ request.wanted_currency }}</div>&ndash;&gt;-->
-<!--        &lt;!&ndash;                <div class="element-2"> {{ request.wanted_type }}</div>&ndash;&gt;-->
-<!--        &lt;!&ndash;                <div class="element-2"> {{ request.wanted_country }} {{ request.wanted_city }}</div>&ndash;&gt;-->
-<!--        &lt;!&ndash;                <div class="element-2"> {{ request.wanted_bank }} {{ request.wanted_purpose }}</div>&ndash;&gt;-->
-<!--        <div class="element-2"><strong> Профит: </strong> {{ request.profit }}</div>-->
-<!--        <div class="element-2"> {{ request.created_on.replace(/T/, ' ').slice(0, -7) }}</div>-->
-
-<!--        <b-button v-if="isNoMaker(request.maker_id)" @click="send(request)" variant="primary">-->
-<!--          Связаться-->
-<!--        </b-button>-->
-<!--        <b-button v-else-if="isLoggedIn" variant="primary" disabled>Моя заявка</b-button>-->
-<!--        <b-button v-else @click="$router.push('/login')" variant="primary">Войти в аккаунт</b-button>-->
-<!--      </v-expansion-panel-content>-->
-<!--    </v-expansion-panel>-->
-<!--  </v-expansion-panels>-->
-
-<!--  <div class="root-element">-->
-<!--    <div class="main-element" v-for="request in requests">-->
-<!--      <b-card bg-variant="light" text-variant="black">-->
-<!--        <b-card>-->
-<!--          <div class="element-1">-->
-<!--            <div class="element-2"><strong> {{ request.id }} </strong></div>-->
-<!--            <div class="element-2" v-if="isCurrentUser(request.maker_username)"> Создатель заявки: Я</div>-->
-<!--            <div class="element-2" v-else> Создатель заявки: {{ request.maker_username }} Рейтинг: {{ request.maker_rank }}</div>-->
-<!--            <div class="element-2"> {{ request.created_on.replace(/T/, ' ').slice(0, -7) }}</div>-->
-<!--          </div>-->
-<!--          <div class="element-1">-->
-<!--            <div class="element-2"><strong>Вы получите: </strong></div>-->
-<!--            <div class="element-2"> {{ request.current_amount }} {{ request.current_currency }}</div>-->
-<!--            <div class="element-2"> {{ request.current_type }}</div>-->
-<!--            <div class="element-2"> {{ request.current_country }} {{ request.current_city }}</div>-->
-<!--            <div class="element-2"> {{ request.current_bank }} {{ request.current_purpose }}</div>-->
-<!--          </div>-->
-<!--          <div class="element-1">-->
-<!--            <div class="element-2"><strong> Вы отдаете: </strong></div>-->
-<!--            <div class="element-2"> {{ request.wanted_amount }} {{ request.wanted_currency }}</div>-->
-<!--            <div class="element-2"> {{ request.wanted_type }}</div>-->
-<!--            <div class="element-2"> {{ request.wanted_country }} {{ request.wanted_city }}</div>-->
-<!--            <div class="element-2"> {{ request.wanted_bank }} {{ request.wanted_purpose }}</div>-->
-<!--          </div>-->
-<!--          <div class="element-2"><strong> Профит: </strong> {{ request.profit }}</div>-->
-<!--        </b-card>-->
-<!--        <b-button v-if="isNoMaker(request.maker_id)" @click="send(request)" variant="primary">-->
-<!--          Связаться-->
-<!--        </b-button>-->
-<!--        <b-button v-else-if="isLoggedIn" variant="primary" disabled>Моя заявка</b-button>-->
-<!--        <b-button v-else @click="$router.push('/login')" variant="primary">Войти в аккаунт</b-button>-->
-<!--      </b-card>-->
-<!--    </div>-->
-<!--  </div>-->
-<!--</template>-->
 <template>
   <v-container fluid>
       <template>
@@ -199,7 +108,6 @@
   data() {
     return {
       requests: [],
-      itemsPerPage: 4,
     }
   },
   computed: {
